srcpkgs/clipmenu/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'clipmenu'
+pkgname=clipmenu
+version=5.6.0
+revision=1
+noarch=yes
+build_style=fetch
```

Comment:
Remove `build_style=` line. Files will be extracted into current dir of do_install.